File: src/main/java/com/codingspezis/android/metalonly/player/plan/PlanRealEntryItem.java
package com.codingspezis.android.metalonly.player.plan;

/**
 * Represents a "real" entry in the plan, i.e. a moderated show
 */
public class PlanRealEntryItem extends PlanItem {

    public PlanRealEntryItem(ShowInformation data) {
        this.data = data;
    }

    @Override
    public boolean isSection() {
        return false;
    }

}

File: src/main/kotlin/com/codingspezis/android/metalonly/player/plan/PlanRealEntryItem.kt
package com.codingspezis.android.metalonly.player.plan

/**
 * Represents a "real" entry in the plan, i.e. a moderated show
 */
class PlanRealEntryItem(data: ShowInformation) : PlanItem() {

    init {
        this.planData = data
    }

    override val isSection: Boolean
        get() = false

}